src/components/Header/Header.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Header = () => <>Hello World!</>;
```

Comment:
Haven't seen these empty tags before. Got a name for them so I can check them out?

Reply:
React fragments! Used for when you need your component to return a single child but don't want to add more nodes to the DOM. 
It's just a shorthand ```<React.Fragment></React.Fragment>```, if you've seen that.